spring-cloud-config-server/pom.xml

```diff
@@ -202,6 +202,18 @@
 			<artifactId>micrometer-tracing-bridge-brave</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>localstack</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<!-- AWS SDK v1 is required by testcontainers-localstack -->
+		<dependency>
+			<groupId>com.amazonaws</groupId>
```

Comment:
Is this OK even though we are using v2?

Reply:
yes, it is ok. Testcontainers Localstack module has a dependency on aws sdk v1 within the container class `LocalStackContainer`. It should be removed in Testcontainers 2.0